README.md

```diff
@@ -14,7 +14,7 @@ Please find the official documentation at: [Vehicle Signal Specification](https:
 To use a specific version of VSS in your toolchain, head over to our [releases page](https://github.com/COVESA/vehicle_signal_specification/releases/).
 
 
-**Latest release:** [2.1](https://github.com/COVESA/vehicle_signal_specification/releases/tag/v2.1)
+[Latest release](https://github.com/COVESA/vehicle_signal_specification/releases/latest)
```

Comment:
Change line 24 below as well. Maybe rephrase it to be more generic so we 

Now:

`Work towards version 2.2 is continuously ongoing`

Easier to maintain:

`Work towards next version is continuously ongoing`

Reply:
good idea! changed it @erikbosch